server/etcdmain/grpc_proxy.go

```diff
@@ -159,6 +163,7 @@ func newGRPCProxyStartCommand() *cobra.Command {
 	// experimental flags
 	cmd.Flags().BoolVar(&grpcProxyEnableOrdering, "experimental-serializable-ordering", false, "Ensure serializable reads have monotonically increasing store revisions across endpoints.")
 	cmd.Flags().StringVar(&grpcProxyLeasing, "experimental-leasing-prefix", "", "leasing metadata prefix for disconnected linearized reads.")
+	cmd.Flags().BoolVar(&grpcProxyEnableLogging, "experimental-enable-grpc-debug", false, "logging all grpc request and response")
```

Comment:
lgtm but we should find a different flag name. There is already a flag named `debug` :) So suggest to change to `experimental-enable-grpc-logging` or something else. Thanks!

Reply:
Thanks @spzala . I agree `--experimental-enable-grpc-logging` makes more sense.

Reply:
Thanks @ahrtr !!

Reply:
Please @stefanmonkey update per  @spzala  so that we can merge this PR.

Reply:
has updated, please check